policy/modules/system/init.if

```diff
@@ -1451,6 +1451,27 @@ interface(`init_relabel_var_lib_dirs',`
 	allow $1 init_var_lib_t:dir relabel_dir_perms;
 ')
 
+########################################
+## <summary>
+##	Create, read, write, and delete the
+##	pseudorandom number generator seed.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`init_manage_random_seed',`
+	gen_require(`
+		type init_random_seed_t;
+	')
+
+	files_search_var_lib($1)
+	files_search_runtime($1)
```

Comment:
You'll need to add a `files_rw_runtime_dirs()` so it can also `add_name` and `remove_name`.